build/scripts/postinstall.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+echo "Promscale installed as a systemd service"
+echo "Enable auto-start with: systemctl enable promscale"
+echo "And start the service now with: systemctl start promscale"
+echo "----------------------------------------"
+CONF_FILE=$(find /etc -name promscale.conf 2>/dev/null | head -n 1)
```

Comment:
why don't we know where this is? How does this work on deb?

Reply:
Config files go into different places in `/etc` folder. I did miss to add the location for deb package, will add.